ford/graphmanager.py

```diff
@@ -76,8 +76,8 @@ def __init__(self, base_url, outdir, graphdir, parentdir, coloured_edges):
         self.types = set()
         self.sourcefiles = set()
         self.blockdata = set()
-        self.graphdir = graphdir
-        self.webdir = base_url + "/" + graphdir
+        self.graphdir = graphdir if graphdir is not None else ""
```

Comment:
Shouldn't that work?
```suggestion
        self.graphdir = graphdir or ""
```

Reply:
Yep, good spot!